python/hail/representation/genomeref.py

```diff
@@ -0,0 +1,167 @@
+from hail.java import Env, handle_py4j, jiterable_to_list
+from hail.typecheck import *
+from hail.representation.interval import Interval
+from hail.representation.variant import Contig
+from hail.utils import wrap_to_list
+
+
+class GenomeReference(object):
+    """An object that represents a genome reference.
+
+    :param str name: Name of reference
+
+    :param contigs: List of contigs
+    :type contigs: list of :class:`.Contig`
+
+    :param x_contig_names: Names of contigs to be treated as X chromosomes
```

Comment:
Oops, I stand corrected.

I don't like the names with _names.  You do just use xContigs in Scala.

Hmm.  I still think I like it better this way.  I'd nuke Scala Contig and add contigLength function to the reference.  Put another way, why is length on Contig but all the other functions are on GenomeReference?  Why not Contig.isX?